__device-tests__/helpers/test-data.js

```diff
@@ -3,7 +3,7 @@ exports.shortText = `Rock music approaches at high velocity.`;
 exports.mediumText = `The finer continuum interprets the polynomial rabbit. When can the geology cheat? An astronomer runs. Should a communist consent?`;
 
 exports.longText = `Beneath the busy continuum blinks the ineffective husband. Why a metric bow outside the official subway? How can the prompt crop exhaust his tree 
-Does this chord crowd my emptied search? A theory bubbles under the cartoon. The discontinued speaker cracks every thick epic. Its extraordinary twin shifts behind
+Does this chord crowd my emptied search? A theory bubbles under the cartoon. The discontinued speaker cracks every thick epic. extraordinary twin shifts behind
```

Comment:
Was this change intentional? If so, I'm just curious to learn why it was needed.

Reply:
The test using the text was being auto corrected otherwise, I saw this as a simple solution.